velox/dwio/dwrf/writer/StatisticsBuilder.h

```diff
@@ -90,10 +98,16 @@ struct StatisticsBuilderOptions {
 /*
  * Base class for stats builder. Stats builder is used in writer and file merge
  * to collect and merge stats.
+ * It can also be used for gathering stats in ad hoc sampling. In this case it
+ * may also count distinct values if enabled in 'options'.
  */
 class StatisticsBuilder : public virtual dwio::common::ColumnStatistics {
  public:
-  explicit StatisticsBuilder(const StatisticsBuilderOptions& options)
+  /// Constructs with 'options'. If 'options' enable count distinct and
+  /// 'disableNumDistinct' is true, distinct values will not be counted.
+  explicit StatisticsBuilder(
+      const StatisticsBuilderOptions& options,
+      bool disableNumDistinct = false)
```

Comment:
Why we want to disable num distinct but options set it? And I don't see it has been used. Thanks!